ui/src/views/indexed-nfa/fragments/aside/aside-tabs.fragment.tsx

```diff
@@ -0,0 +1,92 @@
+import { useMemo, useState } from 'react';
+
+import { Flex } from '@/components';
+import { getRepositoryFromURL } from '@/utils';
+
+import { IndexedNFA } from '../../indexed-nfa.context';
+import { IndexedNFAStyles as S } from '../../indexed-nfa.styles';
+import { Tab, TabContainer } from '../../tabs';
+
+const OverviewFragment: React.FC = () => {
+  const { nfa } = IndexedNFA.useContext();
+
+  return (
+    <S.Aside.Overview.Container
+      css={{ gap: '$4h', p: '$5 $6', height: '336px' }}
+    >
+      <S.Aside.Overview.Row.Container>
+        <S.Aside.Overview.Row.Label>Token ID</S.Aside.Overview.Row.Label>
+        <S.Aside.Overview.Row.Value>{nfa.tokenId}</S.Aside.Overview.Row.Value>
+      </S.Aside.Overview.Row.Container>
+      <S.Aside.Divider.Line />
+      <S.Aside.Overview.Row.Container>
+        <S.Aside.Overview.Row.Label>Network</S.Aside.Overview.Row.Label>
+        <S.Aside.Overview.Row.Value>Mainnet</S.Aside.Overview.Row.Value>
+      </S.Aside.Overview.Row.Container>
+      <S.Aside.Divider.Line />
+      <S.Aside.Overview.Row.Container>
+        <S.Aside.Overview.Row.Label>Standard</S.Aside.Overview.Row.Label>
+        <S.Aside.Overview.Row.Value>ERC_721</S.Aside.Overview.Row.Value>
+      </S.Aside.Overview.Row.Container>
+      <S.Aside.Divider.Line />
+      <S.Aside.Overview.Row.Container>
+        <S.Aside.Overview.Row.Label>Description</S.Aside.Overview.Row.Label>
+      </S.Aside.Overview.Row.Container>
+      <S.Aside.Overview.Description css={{ overflowY: 'scroll' }}>
+        {nfa.description}
+      </S.Aside.Overview.Description>

```

Comment:
Should the `css` prop here be inside the element styling. Also using overflow `auto` and set the `max-height` will show the scroll just when is needed.